### DISCUSSION TITLE
[Projects Beta] Inline project issue UI doesn't show "mentioned this issue" and other system messages that the full issue UI does

### DISCUSSION THREAD

**Discussion by archon810** [community/community#20955] [Projects and Issues]
When you mention or link an issue in the full issue UI, you get helpful system messages, like this: "Bla linked a pull request [N hours ago] #123 that will close this issue."

![image](https://user-images.githubusercontent.com/469543/180057908-6bd61cfb-f462-4366-bd6b-f224eb8bbb75.png)

However, the modal that opens when you click an issue in Projects Beta is inexplicably missing these messages, which creates confusion. I just asked a developer why he didn't link a PR when in reality he did, and the in-project issue UI simply didn't show it.
![image](https://user-images.githubusercontent.com/469543/180058250-abae0fe6-1d07-4d44-b614-271ba85748e6.png)

Please fix this as soon as possible because right now I can't trust the projects issue UI anymore.

**Comment by archon810** [comment]
Any updates please? This is driving us crazy every day.

**Comment by SeBsZ** [comment]
Omg, I thought it was just me. Can't believe this is a bug. Can you please fix?

**Comment by maciejmackowiak** [comment]
Any updates here? 
This created some misunderstandings.....
